### PR TITLE
Upgrade member relations index and add template agenda for founding membership workshop

### PR DESCRIPTION
**Pull request by clausmullie**

### CODE REVIEW

File: activities/member-relations/founding-membership-workshop.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+This is a template agenda for workshops or discussions between potential founding members and the Foundation for Public Code staff
+
+# Agenda
+
+Introduction round (10min)
+
+Background to the Foundation and current status (15min)
+
+Review membership proposal (30min)
+* Foundation governance structure, finacial set up, workflow and 2019 roadmap, high-level overview of potential founding member's codebases for stewardship
```
Comment (ElenaFdR) [inline activities/member-relations/founding-membership-workshop.md]:
```suggestion
* Foundation governance structure, financial set up, workflow, 2019 roadmap and high-level overview of potential founding member's codebases for stewardship
```

File: activities/member-relations/founding-membership-workshop.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+Break (10 min)
+
+Deep dive into Foundation processes (3x30min)
+* [Codebase stewardship process](https://about.publiccode.net/activities/codebase-stewardship/)
+* [Standard for Public Code](http://standard.publiccode.net/)
+* Governance and Financial model
```
Comment (ElenaFdR) [inline activities/member-relations/founding-membership-workshop.md]:
```suggestion
* Foundation governance and financial model
```

File: activities/member-relations/index.md [hunk 3]
```diff
@@ -4,6 +4,37 @@ type: Index
 
 These are all the activities, documents and templates for member relations:
```
Comment (ElenaFdR) [inline activities/member-relations/index.md]:
```suggestion
```

File: activities/member-relations/index.md [hunk 4]
```diff
@@ -4,6 +4,37 @@ type: Index
...
+## 2. Founding membership proposal
+
+At this stage, the Foundation and the public organization work together to identify:
+* appropriate codebases for Foundation codebase auditing, certification and stewardship activities
+* next steps in the collaboration
+* timelines and milestones for 2019
+
+This stage is over when both parties agree on the '[Founding Membership Proposal](founding-membership-proposal.md)'.
+
+This stage may include a '[founding membership workshop}](founding-membership-workshop.md)'.
```
Comment (ElenaFdR) [inline activities/member-relations/index.md]:
```suggestion
This stage may include a [founding membership workshop](founding-membership-workshop.md).
```

File: activities/member-relations/index.md [hunk 5]
```diff
@@ -4,6 +4,37 @@ type: Index
...
+At this stage, the public organization confirms their interest in becoming a member.
+
+This stage is over when the public organization signs the '[Expression of Interest](expression-of-interest.md)'.
+
+## 2. Founding membership proposal
+
+At this stage, the Foundation and the public organization work together to identify:
+* appropriate codebases for Foundation codebase auditing, certification and stewardship activities
+* next steps in the collaboration
+* timelines and milestones for 2019
```
Comment (ElenaFdR) [inline activities/member-relations/index.md]:
```suggestion
* timeline and milestones for 2019
```

File: activities/member-relations/index.md [hunk 6]
```diff
@@ -4,6 +4,37 @@ type: Index
...
+
+This stage may include a '[founding membership workshop}](founding-membership-workshop.md)'.
+
+## 3. Founding membership contract
+
+At this stage, the Foundation and the public organization sign a legally binding contract, including a financial settlement and the founding membership proposal (attached as an annex).
+
+This stage is over when both parties sign a founding membership contract.
+
+> Founding membership contracts are only available to founding members in 2019. From 2020, all members will have an annual membership contracts.
```
Comment (ElenaFdR) [inline activities/member-relations/index.md]:
```suggestion
```

File: activities/member-relations/index.md [hunk 7]
```diff
@@ -4,6 +4,37 @@ type: Index
...
+* appropriate codebases for Foundation codebase auditing, certification and stewardship activities
+* next steps in the collaboration
+* timelines and milestones for 2019
+
+This stage is over when both parties agree on the '[Founding Membership Proposal](founding-membership-proposal.md)'.
+
+This stage may include a '[founding membership workshop}](founding-membership-workshop.md)'.
+
+## 3. Founding membership contract
+
```
Comment (ElenaFdR) [inline activities/member-relations/index.md]:
```suggestion

> Founding membership contracts are only available to founding members in 2019. From 2020, all members will have an annual membership contracts.
```

**Comment by clausmullie** [comment]
thanks @ElenaFdR for your review! :)

**Pull request merged**